html/viewer/js/app.js

```diff
@@ -74,15 +79,30 @@ var IconViewer = {
       },
       parent: container
     });
+    for (let platform in icon.source) {
+      iconContainer.dataset[`uri_${platform}`] = IconList.getFullIconURI(icon, new Map([[platform, true]]));
```

Comment:
It might be easier to hold a reference to the `icon` object on the element rather than having all those data attributes.
`iconContainer.iconData = icon`.

Reply:
@nt1m can you contribute to this PR? We would like to ship by the end of this week, thanks!